packages/eslint-plugin-react-hooks/__tests__/ESLintRulesOfHooks-test.js

```diff
@@ -432,6 +431,7 @@ const tests = {
       `,
       errors: [
         topLevelError('Hook.useState'),
+        topLevelError('Hook._useState'),
```

Comment:
We could disallow this by only allowing the leading underscore for top level identifiers, but figured it might be nice to recognize `Hook.__useSemiPrivateHook`?